shapeshift/web
# feat: use the official gnosis network icon

## Description

Use the Official Gnosis logo for asset icons and network icon. xDAI remains the fee currency and keeps its specific logo.



## Pull Request Type


## Issue (if applicable)



closes #

## Risk
None.



## Testing
- Check that the assets that are on Gnosis and which have network icons in certain interfaces (Assets page, Assets dropdowns) have the proper Gnosis icon.
- Check that the Gnosis network in the network selection menu with the non-native wallets (e.g. MetaMask) show the proper icon.



### Engineering
☝️ 

Note: There will be some overlap with the changes in #4641 it also seem some of the Cosmos URL for icons get updated after each run of `yarn generate::asset-data`, not sure which ones should be used, they both seem to work, as well as the previous ones.



### Operations
☝️ 


## Screenshots (if applicable)
Before

![image](https://github.com/shapeshift/web/assets/88804546/f6c96567-21cc-43e6-8a56-20b5bb967b89)
![image](https://github.com/shapeshift/web/assets/88804546/80961d7c-be84-4f60-83f6-98144ce303c8)


After

![image](https://github.com/shapeshift/web/assets/88804546/7babf72b-72f5-499b-9437-1912ac61e770)
![image](https://github.com/shapeshift/web/assets/88804546/d66a0d3e-5942-4b2e-ae45-5f5ad2f75bce)
